api/ApplicationSettings.md

```diff
@@ -57,18 +55,6 @@ to Chromium Preferences.
 ## Settings
 
 
-### accept_language_list
```

Comment:
When API changes we should update the Migration Guide document:
https://github.com/cztomczak/cefpython/blob/master/docs/Migration-guide.md